src/hotspot/cpu/aarch64/aarch64_vector_ad.m4

```diff
@@ -215,6 +215,26 @@ source %{
           return false;
         }
         break;
+      case Op_AddVHF:
+      case Op_SubVHF:
+      case Op_MulVHF:
+      case Op_DivVHF:
+      case Op_MinVHF:
+      case Op_MaxVHF:
+      case Op_SqrtVHF:
+        // FEAT_FP16 is enabled if both "fphp" and "asimdhp" features are supported.
```

Comment:
It's an unary op and we should add it to `is_vector_unary_op_name` in `adlc/dfa.cpp`.
See the related code in the previous patch https://github.com/openjdk/jdk/pull/9534